helm/prometheus-rules/templates/alerting-rules/management-cluster.rules.yml

```diff
@@ -128,3 +128,17 @@ spec:
         severity: notify
         team: ludacris
         topic: kubernetes
+    - alert: ManagementClusterDeploymentMissing
+      annotations:
+        description: '{{`Deployment {{ $labels.workload_name }} is missing.`}}'
+        opsrecipe: management-cluster-deployment-is-missing/
+      expr: absent(kube_deployment_status_condition{namespace="giantswarm", condition="Available", workload_name="management-cluster-admission-controller-manager"})
```

Comment:
Is the name of the deployment actually `management-cluster-admission-controller-manager`? It does not ring any bell for me right now.

Reply:
Unless I've got the wrong deployment. There definitely is a deployment called `management-cluster-admission-controller-manager` on gauss

Reply:
Ah okay. :) I guess it is not owned by firecracker. Maybe for now we can try with just the `aws-admission-controller`.

Reply:
Isn't it going to become owned by Phoenix though going forward? (Might be wrong)

Either way, happy to reduce the scope of this for now to just focus on the aws-admission-controller and then adding more when needed.